taglib/riff/wav/wavfile.cpp

```diff
@@ -47,7 +48,8 @@ class RIFF::WAV::File::FilePrivate
     properties(0),
     tagChunkID("ID3 "),
     hasID3v2(false),
-    hasInfo(false)
+    hasInfo(false),
+    acidInfo(NULL)
```

Comment:
It's actually a style issue, not a compatibility issue.  As a general rule of thumb, C uses `NULL`, C++ uses `0` and C++11 uses `nullptr`.